src/main/java/seedu/nutrigoals/logic/commands/AddCommand.java

```diff
@@ -16,11 +16,11 @@ public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a food to the tracker. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a food to the tracker. \n"
```

Comment:
Would it be better to say `Adds a food to NutriGoals` instead to be consistent with `ExitCommand`'s message usage?